components/molecules/AuthSection/auth-section.tsx

```diff
@@ -110,6 +111,15 @@ const AuthSection: React.FC = ({}) => {
         <FiLogOut className="group-hover:text-light-orange-10" />
         <Text className="group-hover:text-light-orange-10">Disconnect</Text>
       </span>,
+      <Link
+        href={"https://docs.opensauced.pizza/community/faqs/"}
```

Comment:
```suggestion
        href={"https://docs.opensauced.pizza/community/"}
```
Let's link here in stead.

Reply:
@bdougie actually this URL gives a 404 not found page, should I link to the welcome-to-community page instead
https://docs.opensauced.pizza/community/welcome-to-the-community/

Reply:
Ok perhaps we change the title to FAQs then until we have a better options